nncf/torch/quantization/default_quantization.py

```diff
@@ -31,20 +31,11 @@
         operator_metatypes.PTDepthwiseConv3dSubtype,
```

Comment:
What about the rest of the `default_quantization.py` files across the frameworks?

Reply:
Other frameworks are outside of the scope of this PR, the only purpose of this pr is to align OV and Torch backends. I suggest to leave this for future PRs, this PR should be merged till next Monday to make progress for Torch PTQ